### PR TITLE
Related resources

### LINKED ISSUE #516: Create the front matter for the related info URLS
Related info  frontmatter urls slugs will be used to store the URLS for issue #515 

Perhaps we should limit this to 5? not sure if we can enforce logic in front matter, but we can add that the the contributing guide.

### LINKED ISSUE #517: Document how related URL front matter works in the contributor guide.
The contributor guide should include a section on how to set related URLs in the front matter.

### PR DESCRIPTION
**Pull request by jerelmiller**
Closes #516 
Closes #517 

## Description
Adds the ability to specify related resources for a guide in the frontmatter of the MDX document. 

## Screenshot(s)

Screen Shot 2020-07-31 at 11 41 09 PM

### CODE REVIEW

File: src/components/Tag.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import PropTypes from 'prop-types';
+import { css } from '@emotion/core';
+import { Link } from 'gatsby';
+import styled from '@emotion/styled';
+import { rgba } from 'polished';
+
+const isInteractive = ({ as, interactive }) =>
+  interactive || as === 'a' || as === Link;
+
+const Tag = styled.span`
```
Comment (jerelmiller) [inline src/components/Tag.js:10]: TODO: Need to update to use the gatsby theme tag component once a new version is released.

Comment (zstix) [inline src/components/Tag.js:10]: Can we use the component you just merged into the `@newrelic/gatsby-theme-newrelic`?

Comment (zstix) [reply src/components/Tag.js:10]: Got, saw your comment after I reviewed. I'll handle this for ya

File: src/@newrelic/gatsby-theme-newrelic/icons/feather.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import featherIcons from '@newrelic/gatsby-theme-newrelic/src/icons/feather';
+
+export default {
+  ...featherIcons,
+  'external-link': (
```
Comment (LizBaker) [inline src/@newrelic/gatsby-theme-newrelic/icons/feather.js:6]: do we want to add this icon to the theme or is it more of a one-off for the dev site?

Comment (zstix) [reply src/@newrelic/gatsby-theme-newrelic/icons/feather.js:6]: I was wondering the same thing. I suspect that was the intention (thus the file name).

This seems like something that would be helpful across a few sites. I'll make another PR / release for this shortly.

**Pull request merged**